exporter/opentelemetry-exporter-prometheus/tests/test_prometheus_exporter.py

```diff
@@ -167,3 +167,30 @@ def test_sanitize(self):
         self.assertEqual(collector._sanitize(",./?;:[]{}"), "__________")
         self.assertEqual(collector._sanitize("TestString"), "TestString")
         self.assertEqual(collector._sanitize("aAbBcC_12_oi"), "aAbBcC_12_oi")
+
+    def test_list_labels(self):
```

Comment:
It might be cleaner to test this by moving the mapping code into a helper function and adding a bunch of test cases (similar to `_sanitize()`)

Reply:
All right, refactoring...

Reply:
fixed